test/active_model_serializers/model_test.rb

```diff
@@ -18,5 +18,46 @@ def test_initialization_with_string_keys
 
       assert_equal model_instance.read_attribute_for_serialization(:key), value
     end
+
+    def test_attributes_can_be_read_for_serialization
```

Comment:
It looks like this test is about a change via accessor instead of constructor. Maybe the test name could be a little more descriptive in that regard?